docs/source/tutorials/pasha/scripts/launch_pasha_nasbench201.py

```diff
@@ -1,21 +1,18 @@
-# PASHA: Efficient HPO with Progressive Resource Allocation
+# PASHA: Efficient HPO and NAS with Progressive Resource Allocation
 
+import logging
+
+import pandas as pd
+from benchmarking.commons.benchmark_definitions.nas201 import nas201_benchmark
+from syne_tune.backend.simulator_backend.simulator_callback import \
+    SimulatorCallback
+from syne_tune.blackbox_repository import BlackboxRepositoryBackend
+from syne_tune.blackbox_repository.repository import load_blackbox
 from syne_tune.experiments import load_experiment
-from syne_tune.stopping_criterion import StoppingCriterion
-from syne_tune.tuner import Tuner
 from syne_tune.optimizer.baselines import baselines_dict
 from syne_tune.optimizer.schedulers.hyperband import HyperbandScheduler
-from syne_tune.backend.simulator_backend.simulator_callback import SimulatorCallback
-from benchmarking.definitions.definition_nasbench201 import (
-    nasbench201_benchmark,
-    nasbench201_default_params,
-)
-from benchmarking.blackbox_repository.simulated_tabular_backend import (
-    BlackboxRepositoryBackend,
-)
-from benchmarking.blackbox_repository import load
-import pandas as pd
-import logging
+from syne_tune.stopping_criterion import StoppingCriterion
+from syne_tune.tuner import Tuner
 
 
 def load_nb201(dataset_name, nb201_random_seed):
```

Comment:
Could you please clean up this example? You can have a look at how the other examples are done, for example `examples/launch_nasbench201_simulated.py`.

This `load_nb201` is not needed, so please remove it.

I am not keen on having code in here that is hard to maintain, so please just keep it simple and looking like the other examples.

Reply:
I've simplified this tutorial now, thanks for the suggestion